modules/components/public/themeStyles/beagle/beagle.css

```diff
@@ -430,16 +430,38 @@ div {
 /**************************************/
 /******* styling for match box  *******/
 /**************************************/
+.match-box {
+  display: flex;
+  flex-direction: column;
+}
 .match-box-id-form > * {
   margin-bottom: 1.5em;
 }
 .match-box-id-form textarea {
   min-height: 100px;
 }
-.match-box-select-entity-form,
-.match-box-results-table {
+.match-box-select-entity-form {
   margin-bottom: 20px;
 }
+.match-box-results-table {
+  display: flex;
+  flex-direction: column;
+}
+
+.match-box .tabs {
+  display: flex;
+  flex-direction: column;
+}
+.match-box .tabs .tabs-content {
+  display: flex;
+  flex-direction: column;
+}
+.match-box .tabs .tabs-titles {
+  display: block;
+}
+.match-box .tabs .tabs-titles .tabs-title {
+  float: left;
+}
```

Comment:
aside from the margin change I'm not sure if these changes are beagle specific, kind of seems like a global match-box layout thing. But I could be wrong (maybe it is specific), what do you think?

Reply:
yeah we can move this into matchbox itself